### DISCUSSION TITLE
Client  with virtual IP cannot receive replies from the server side

### DISCUSSION THREAD

**Discussion by rootBiao** [strongswan/strongswan#1872] [Q&A]
Configure a virtual IP address pool (12.0.0.0/8) on the server side.
The client  got  a virtual IP (12.0.0.1)  from server.
But client  with virtual IP cannot receive replies from the server side.

![image](https://github.com/strongswan/strongswan/assets/87700793/461f266a-80f1-4564-b5fc-d85e19e108ab)
![image](https://github.com/strongswan/strongswan/assets/87700793/68eba990-b139-4460-8096-b69b6a4b3247)
![image](https://github.com/strongswan/strongswan/assets/87700793/779fcf95-f6c4-41b6-9b8c-fc07dd24c5a5)

**Comment by tobiasbrunner** [comment]
Please read the [docs regarding forwarding traffic](https://docs.strongswan.org/docs/5.9/howtos/forwarding.html).

**Comment by rootBiao** [comment]
这是来自QQ邮箱的假期自动回复邮件。你的邮件，陈陆彪已经收到。
